getting-started/gopher-run/cmd/main.go

```diff
@@ -0,0 +1,133 @@
+package main
```

Comment:
Please add a package comment.

https://github.com/golang/go/wiki/CodeReviewComments#package-comments

```
// The cmd command starts a Gopher Run game server.
package main

...
```

(Newline between the license and package comment and no newline between comment and package declaration line.)

Reply:
Done